internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -650,7 +651,7 @@ func (e *Etcd) argsForControlPlane(ctx context.Context, r runtime.Runtime) error
 
 // recoverFromSnapshot recovers etcd data directory from the snapshot uploaded previously.
 func (e *Etcd) recoverFromSnapshot(hostname, primaryAddr string) error {
-	manager := snapshot.NewV3(nil)
+	manager := snapshot.NewV3(logging.Wrap(log.Writer()))
```

Comment:
`log.Writer` on purpose?, the others used `os.Stderr`

Reply:
yep, this one is _inside_ machined, so it should log properly to the console (`log.Writer` is set up for that)

other uses are in the CLI